commands/management/add_mysql_test.go

```diff
@@ -147,6 +148,23 @@ Table statistics collection disabled (always).
 	})
 }
 
+func TestCredentials(t *testing.T) {
+	data := `{"username": "testuser", "password": "testpass", "agentpassword": "testagentpass"}`
+	creds := commands.Credentials{}
+	cr, _ := CreateDummyCredentialsFile(data, "json", false)
+	ce, _ := CreateDummyCredentialsExecutable(data)
+
+	if err := creds.ReadFromSource(cr); err != nil {
+		t.Fatalf("failed to read from source: %v", err)
+	}
```

Comment:
we use `github.com/stretchr/testify/assert` or `github.com/stretchr/testify/require` to assert in tests.